packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -16,14 +16,18 @@ import './index.scss';
 import { config } from '../../lib/config';
 
 export type NewUserEmailFormProps = {
-  getString: (id: string) => string;
+  getString?: (id: string) => string;
+  setValidEmailSet: Function;
+  setAccountExists: Function;
```

Comment:
If we know the types of these functions, would be good to declare them, and not just use `Function`.

Reply:
I updated these types locally, I'm getting a typescript error in the test, I'd like to pass the correct thing here to silence the error, but I'm not sure how to fake this type, I haven't found anything terribly useful in my searches yet...

Reply:
I actually fixed this up by just including `setState` calls in the stories and tests, it works and it's more true to the operating state of the component 👍